src/GafferSceneUI/Inspector.cpp

```diff
@@ -291,14 +296,25 @@ void Inspector::inspectHistoryWalk( const GafferScene::SceneAlgo::History *histo
 						if( nonEditableReason.empty() )
 						{
 							result->m_editFunction = [source = result->m_source] ( bool unused ) { return source; };
+							result->m_disableEditFunction = disableEditFunction( result->m_source.get(), history );
 							result->m_editWarning = editWarning;
 						}
 						else
 						{
 							result->m_editFunction = nonEditableReason;
+							result->m_disableEditFunction = nonEditableReason;
 						}
-
-						result->m_disableEditFunction = disableEditFunction( result->m_source.get(), history );
+					}
+					else if( node->ancestor<EditScope>() && node->ancestor<EditScope>() != result->m_editScope )
+					{
+						result->m_disableEditFunction = fmt::format(
+							"Edit authored in EditScope ({}), set it as the target EditScope to disable.",
+							node->ancestor<EditScope>()->relativeName( node->ancestor<EditScope>()->scriptNode() )
+						);
+					}
+					else if( !node->ancestor<EditScope>() && result->m_editScope )
+					{
+						result->m_disableEditFunction = "Edit was not authored in any EditScope, set target EditScope to (None) to disable.";
```

Comment:
If we went with my suggestion above, I think this could follow the same form :

> Edit is not in the current edit scope. Change scope to None to disable.

Reply:
Switched over to the suggested `Edit is not in the current edit scope. Change scope to None to disable.`